Nexar.Supply.Xll/NexarQueryManager.cs

```diff
@@ -35,6 +35,14 @@ public class NexarQueryManager
         public string NexarClientId { get; set; }
         public string NexarClientSecret { get; set; }
 
+        /// <summary>
+        /// Whether or not to include specific features when querying on behalf of the client.
+        /// A client may not have access if they are a self-serve customer.
+        /// </summary>
+        public bool IncludeDatasheets { get; set; } = true;
```

Comment:
Boolean options defaulting to true are normally counter intuitive.
I'd turn this into `ExcludeDatasheets` with expected default false and same default effect.